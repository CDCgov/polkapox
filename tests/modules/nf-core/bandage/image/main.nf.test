nextflow_process {

    name "Test Process BANDAGE"
    script "modules/nf-core/bandage/image/main.nf"
    process "BANDAGE"

    test("Bandage/Image Test") {

        when {
            params {
                
            }
            process {
                """
               input[0] = Channel.of([
                    [ id: 'test_sub'],
                    [
                        file("${projectDir}/tests/nf-test-data/test_sub.bridges_applied.gfa", checkIfExists: true)
                    ]
                ])
                """
            }
        }

        then {
            assert process.success
            assert snapshot(process.out).match()
        }

    }

}
