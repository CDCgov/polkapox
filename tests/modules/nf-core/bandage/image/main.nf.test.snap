{
    "Bandage/Image Test": {
        "content": [
            {
                "0": [
                    
                ],
                "1": [
                    
                ],
                "2": [
                    
                ],
                "png": [
                    
                ],
                "svg": [
                    
                ],
                "versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-07-07T14:35:15.734506768"
    }
}